src/Microsoft.Extensions.Logging.Console/ConsoleLoggerFactoryExtensions.cs

```diff
@@ -93,6 +122,23 @@ public static ILoggerFactory AddConsole(this ILoggerFactory factory, LogLevel mi
             return factory;
         }
 
+        /// <summary>
+        /// Add a <see cref="ConsoleLoggerProvider"/> configured with <see cref="ConfigurationConsoleLoggerSettings"/> 
+        /// taken from the provided <see cref="IConfiguration"/>, e.g. from a settings file. 
```

Comment:
Avoid Latin abbreviations, e.g. `e.g.`, and use the English wording, i.e. `some animals are cute, for example, a kitty cat.`